packages/eas-cli/src/build/local.ts

```diff
@@ -8,8 +8,14 @@ import { ora } from '../ora';
 const PLUGIN_PACKAGE_NAME = 'eas-cli-local-build-plugin';
 const PLUGIN_PACKAGE_VERSION = '0.0.120';
 
+export enum LocalBuildMode {
+  DISABLED,
```

Comment:
I mostly think it's confusing because this is unexpected default. Local builds are not the default way of building so "disabling" them sounds weird. They should be "enabled" when needed.